common/configurable/src/test/java/io/helidon/common/configurable/ThreadPoolTest.java

```diff
@@ -279,40 +276,6 @@ void testNullRejectionPolicy() {
         ), "rejectionPolicy is null");
     }
 
-    @Test
```

Comment:
Please return the test, it will work when you run it from Maven.

Reply:
So the problem comes from compiler plugin, as the tests seem to be compile with module path.
To fix the problem, add this section to `maven-compiler-plugin` in `pom.xml` of the module:
```
                <executions>
                    <execution>
                        <id>default-testCompile</id>
                        <configuration>
                            <!--
                            We use JUL from tests, need this for compilation
                             -->
                            <compilerArgs>
                                <compilerArg>--add-modules</compilerArg>
                                <compilerArg>java.logging</compilerArg>
                            </compilerArgs>
                        </configuration>
                    </execution>
                </executions>
```

and re-introduce the test as it was